### ISSUE TITLE
Only GeoLite2 product is now working with geoipupdate

### ISSUE THREAD

**Issue by allaire** [Temelio/ansible-role-maxmind-geoip#56]
I think the legacy packages and other ids were removed, because I was always getting an error:

https://updates.maxmind.com/app/update_secure?db_md5=00000000000000000000000000000000&challenge_md5=9481d0dadaec12b8a3e9b8ea475a3d08&user_id=999999&edition_id=506

I think we should only have now:

```
    maxmind_geoip_geoipupdate_product_ids:
      - "GeoLite2-City"
      - "GeoLite2-Country"
```

**Comment by allaire** [comment]
Also now files in `files-names` are not valid anymore, and now the extension is only `.gz`, and it crashes when it tries to unarchive it.

**Comment by achaussier** [comment]
Hello,

Thanks for this issue, I update the role now to remove legacy databases and update configuration keys.

But about your second problem, I've test items in "file_names" and they're valid.
If you always have this issue can you open a new issue and join a log extract please.

Have a nice day,

Alexandre

**Comment by allaire** [comment]
@achaussier Thanks for the fix.

About the file extensions, I'm talking about http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz

See the table here: https://dev.maxmind.com/geoip/legacy/geolite/ — Most download links are dat.gz extension, and the role fail to extract those. It would be interesting to support both tar.gz and .gz (related? https://github.com/ansible/ansible-modules-core/issues/3241)

**Comment by achaussier** [comment]
Thanks, I had the same error after the update. I've created a workaround for .gz files available in 0.3.1 release.
